crates/txtx-core/src/lib.rs

```diff
@@ -279,15 +263,23 @@ pub async fn start_supervised_runbook_runloop(
             ActionItemResponseType::ValidateBlock => {
                 // Handle background tasks
                 if !background_tasks_futures.is_empty() {
+                    // todo: we still need to figure out how to have multiple flows in a supervised execution
```

Comment:
This was an existing issue that I discovered here. Added to planner [here](https://teams.microsoft.com/l/entity/com.microsoft.teamspace.tab.planner/mytasks?tenantId=0f1e1f86-ff36-4233-ba7c-e2d48a07bb41&webUrl=https%3A%2F%2Ftasks.teams.microsoft.com%2Fteamsui%2FpersonalApp%2Falltasklists&context=%7B%22subEntityId%22%3A%22%2Fv1%2Fpremiumplan%2Fc7a7d962-0836-4ed0-8a67-774535750432%2Forg%2Fdeb736b0-d91b-ef11-9f85-000d3a537223%2Fview%2Fboard%2Ftask%2F9E126BEB-DF7D-4F10-B109-D88EC5DFA53D%22%7D)